### DISCUSSION TITLE
I'm retiring the Weird Wide Webring...

### DISCUSSION THREAD

**Discussion by jackmcdade** [jackmcdade/weird-wide-webring#74] [Announcements]
...at least in its current form. Here's a list of things I'm not super happy with.

- Editing a JSON file on GitHub has been problematic UX and has led to issues and downtime when things aren't properly formatted. 
- Creating links in a flat list where the newest is on the bottom doesn't really help anyone with discovery
- The types of sites we get submissions for weren't exactly what I was envisioning — which is fine — but everything is so super niche that the odds of someone spending time on the next link is pretty low. We owe the visitors of a webring a bit more than a bounce.
- There's no checks in place to make sure that sites are active or haven't been replaced with spam/p0rn/etc
- Probably a lot more

Thank you everyone for going on this ride, it was nostalgic. Maybe I can bring it back to life someday in the future when I have a better idea how to make it something people can be excited about. If you have ideas, feel free to discuss them below!

I'm will eventually archive the repo for now i'll leave it open in case anyone wants to discuss this. Everything should still work for the foreseeable future so people's sites don't have dead links on them.

**Comment by ghost** [comment]
rip

**Comment by anna-filou** [comment]
It’s a shame but I understand your reasons. Still, the internet _does_ need more weird, and more ways of discovering it. In time, I know we’ll figure out a way. 🧡

**Comment by ghost** [comment]
Just add specify more how to build weird websites. Not all devs know how to code "weird"
